### ISSUE TITLE
User: Feedback Form: Submit Feedback: Automatically generate github issue

### ISSUE THREAD

**Issue by Chingujo** [department-of-veterans-affairs/caseflow-feedback#71]
As a Product Support Analyst, I would like to have a github issue automatically created ever time a Feedback comment is submitted so that I can view it in my Appeals-Support repository for processing.

# Acceptance Criteria 
1. Verify that after clicking on the **_Send Feedback_** button, a the github new issue is automatically generated in the Appeals-support repository.
1. Verify that the new issue uses the support [Issue Template](https://github.com/department-of-veterans-affairs/appeals-support/blob/master/ISSUE_TEMPLATE.md).
1. Verify that for the new issue, the **Reporter Information** section has the following fields completed containing the appropriated data:
     - Email Address - contains submitter's email address
1. Verify that for the new issue, the **Issue Description** section has the following fields completed containing the appropriated data:
     - Details - contains the Feedback comment
1. Verify that for the new github issue, the following labels have been added:
     - Product Support Team
     - Source - Feedback
     - Current Sprint
     - [Application Label]:  **_Certification, Dispatch, eFolder_**
1. Verify that in the Dashboard section on the Feedback Admin page, the _"Github Issue"_ displays the newly generated github issue number. 

## Mockups 
**_View - User Feedback Form - Fields designated for Github_**
![image](https://cloud.githubusercontent.com/assets/23063044/22076479/3b0f545e-dd7e-11e6-8371-7d9c279cb808.png)

**_View - Github Issue - Template Sample_**
![image](https://cloud.githubusercontent.com/assets/23063044/22037358/d4dfddca-dcc4-11e6-89f5-ea6a1d6f1645.png)
# Technical Notes

- Github credentials need - caseflow developer (https://github.com/department-of-veterans-affairs/caseflow-developer)
- Need keys 
- Will handle error situation in another Story - unable to create github issue, slack without github issue, updating feedback with github issue at later time.

# Related Stories 
Epic: Feedback Admin View: Ability to view and report incoming feedback form Caseflow - [#1050](https://github.com/department-of-veterans-affairs/appeals-pm/issues/1050)

**Comment by nikitarockz** [comment]
# My Feedback
### Sections with Suggested Changes
- [X] Title - Done
- [X] User Story - refer to Changes section
- [X] Acceptance Criteria - Refer to changes section
- [X] Mockup - @Chingujo needs updating and refer to Changes section
- [ ] Related Stories

### Comments
n/a
### Questions
n/a

### Changes
As a Product Support Analyst, I want to click a button that opens a "New Issue" template in a separate tab so I can quickly and efficiently catalog important issues as I move through feedback forms. 

# Acceptance Criteria 
1. Verify that after clicking on the Create Github Issue button on the Feedback Details modal, a Github "New Issue" template is displayed in a separate browser tab.
1. Verify that after clicking on the Create Github Issue button on the Feedback Details modal, a the new issue is open in the Appeals-pm repository.
1. Verify that after clicking on the **_Submit new issue_** button for the Github Issue, the _"Github Issue"_ value is automatically updated with the new issue number for that issue in the Dashboard section on the Feedback Admin page.

**Comment by anyakhvost** [comment]
In demo, it will not create an actual Github issue. This can only be fully tested in UAT. Waiting on @ariperez to create a support user and generate the access_token for UAT

**Comment by astewarttistatech** [comment]
Marked as blocked due to @aroltsch 's comment. (see above)

**Comment by astewarttistatech** [comment]
**PASSED**

Steps to Validate: 
1. Go to feedback (UAT) from any product or directly
2. Fill out text fields
3. Submit Feedback
4. Go to https://feedback.cf.uat.ds.va.gov/admin
5. Notice that the feedback submitted now has been assigned a github issue. 

Screenshot: 
<img width="944" alt="screen shot 2017-01-25 at 11 58 38 am" src="https://cloud.githubusercontent.com/assets/23038505/22301601/988b6976-e2f9-11e6-8fd7-29c7a0393fe7.png">
